apps/web/src/components/AssessmentStepTracker/utils.ts

```diff
@@ -451,3 +426,72 @@ export const groupPoolCandidatesByStep = (
 
   return stepsWithGroupedCandidates;
 };
+
+export type ResultFilters = {
+  query: string;
+  [NO_DECISION]: boolean;
+  [AssessmentDecision.Successful]: boolean;
+  [AssessmentDecision.Hold]: boolean;
+  [AssessmentDecision.Unsuccessful]: boolean;
+};
+
+export const defaultFilters: ResultFilters = {
+  query: "",
+  [NO_DECISION]: true,
+  [AssessmentDecision.Successful]: true,
+  [AssessmentDecision.Hold]: true,
+  [AssessmentDecision.Unsuccessful]: true,
+};
+
+const statusIsAvailable = (filter?: boolean) => {
+  return typeof filter !== "undefined" && filter;
+};
```

Comment:
Anywhere we need to check if `filter` is true, couldn't we just use `if (filter)` directly? And it not, then `!!filter`?